src/main.rs

```diff
@@ -51,11 +41,15 @@ use futures::{
     },
     task,
 };
+use lazy_panic::set_panic_message;
+use lazy_static::lazy_static;
 use rocket::{
+    self,
     config::{
         Config,
         Environment,
     },
+    get,
```

Comment:
is this higher up in the list because of alphabetizing? or some other reason?

Reply:
I mostly just ask because i'm used to seeing it closer to `post`, don't really have a preference that it be down there though

Reply:
`rust fmt` sorts the use statements, so yeah, it's purely alphabetical.